app/javascript/gobierto_dashboards/webapp/components/WidgetIndicatorTable.vue

```diff
@@ -89,7 +90,7 @@ export default {
     const [y1, y2, y3, y4] = this.values.sort(
       ({ date: a }, { date: b }) => new Date(a) < new Date(b)
     );
-    const years = [...[y1, y2, y3, y4]];
+    const years = [...[y1, y2, y3, y4]].filter(Boolean);
```

Comment:
What does this filter do? Maybe a comment would be good

Reply:
It's a way to remove `null` or `undefined` from an array. I tested some cases where there weren't four years to parse, but two or three. I'll write it down

Reply:
good to know